test/bigtest/interactors/VoucherEditInteractor.js

```diff
@@ -0,0 +1,25 @@
+import {
+  interactor,
+  isPresent,
+  Interactor,
+} from '@bigtest/interactor';
+
+export default interactor(class EditVoucherView {
+  static defaultScope = '[data-test-edit-voucher-form]';
+
+  voucherNumberInput= isPresent('input[name="voucherNumber"]');
+  disbursementNumberInput = new Interactor('input[name="disbursementNumber"]');
+  disbursementDateInput = new Interactor('input[name="disbursementDate"]');
+  disbursementAmountInput = new Interactor('input[name="disbursementAmount"]');
+  saveFormButton = new Interactor('[data-test-save-button]');

```

Comment:
FYI, we have `ButtonInteractor`, as well as TextfieldInteractor, in stripes-acq-components. However, it's not crucial to use them if you don't need